src/Service.Tests/DatabaseSchema-DwSql.sql

```diff
@@ -235,6 +245,7 @@ VALUES (1, 'Star Trek', 'SciFi', NULL), (2, 'Cinderella', 'Tales', 3001),(3,'Ún
 (5, 'Snow White', 'AnotherTales', 3001);
 
 INSERT INTO [foo].[magazines](id, title, issue_number) VALUES (1, 'Vogue', 1234), (11, 'Sports Illustrated', NULL), (3, 'Fitness', NULL);
+INSERT INTO [bar].[magazines](upc, comic_name, issue) VALUES (0, '0', 0);
```

Comment:
super nit to not just use an int as a sample name.
```suggestion
INSERT INTO [bar].[magazines](upc, comic_name, issue) VALUES (0, 'MyComic', 0);
```